### ISSUE TITLE
Localstack 0.9.5 breaks lambda execution

### ISSUE THREAD

**Issue by alphachai** [localstack/localstack#1382]
Discovered this morning that all my lambda integration tests which use localstack:latest are broken. Downgrading to 0.9.4 seems to fix the problem.

```
E           botocore.exceptions.ClientError: An error occurred (InternalFailure) when calling the Invoke operation (reached max retries: 4): Error executing Lambda function arn:aws:lambda:us-east-1:000000
000000:function:my_lambda: Lambda process returned error status code: 1. Output:
E           Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
E           must specify at least one container source
E           Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running? Traceback (most recent call last):
E             File "/opt/code/localstack/localstack/services/awslambda/lambda_api.py", line 320, in run_lambda
E               event, context=context, version=version, asynchronous=asynchronous)
E             File "/opt/code/localstack/localstack/services/awslambda/lambda_executors.py", line 62, in execute
E               result, log_output = self._execute(func_arn, func_details, event, context, version, asynchronous)
E             File "/opt/code/localstack/localstack/services/awslambda/lambda_executors.py", line 204, in _execute
E               result, log_output = self.run_lambda_executor(cmd, environment, asynchronous)
E             File "/opt/code/localstack/localstack/services/awslambda/lambda_executors.py", line 137, in run_lambda_executor
E               (return_code, log_output))
E           Exception: Lambda process returned error status code: 1. Output:
E           Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
E           must specify at least one container source
E           Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```

**Comment by alphachai** [comment]
I'm using `pytest` with `pytest-localstack` to run my tests. Here's an example of what breaks against latest.

*tests/conftest.py*
```python
localstack = pytest_localstack.patch_fixture(
    services=["kinesis", "lambda"], scope="module", autouse=False, localstack_version="0.9.4"
)

@pytest.fixture(scope="class")
def mock_lambda(environment):
    lambda_client = boto3.client("lambda")
    with open(str(Path().absolute() / "dummy_lambda/dist/build.zip"), "rb") as f:
        zipped_code = f.read()
        lambda_client.create_function(
            FunctionName="my_lambda",
            Runtime="python3.6",
            Role="foobar",
            Handler="main.lambda_handler",
            Code=dict(ZipFile=zipped_code),
            Timeout=300,
            Environment={"Variables": environment(MOCK_AWS="true")},
        )


@pytest.fixture
def invoke_lambda():
    def inv(name="my_lambda", payload):
        client = boto3.client("lambda")
        response = client.invoke(
            FunctionName=name,
            InvocationType="RequestResponse",
            LogType="Tail",
            Payload=json.dumps(payload).encode(),
        )
        body = response["Payload"].read()
        try:
            body = json.loads(body)
        except:
            pass
        return response, body

    return inv
```

*tests/integration/test_lambda.py*
```python
@pytest.mark.postbuild
@pytest.mark.usefixtures("localstack", "kinesis", "mock_lambda")
class TestMockLambda:
    @pytest.mark.parametrize(
        "fixture_name,fixture", [(k, v) for k, v in fixtures.DATA.items()]
    )
    def test_lambda_fixtures(
        self, invoke_lambda, kinesis_payload, fixture_name, fixture
    ):
        logger = ServerlessLogger(level=logging.DEBUG, process="shepherd")
        logger.persist = True
        response, body = invoke_lambda(
            name="my_lambda", payload=kinesis_payload(fixture["payload"])
        )
        utils.emit_logs(body)
        assert response["StatusCode"] // 100 == 2
        assert fixture["response"]["stats"] == body["stats"]
```

**Comment by whummer** [comment]
Thanks for reporting @alphachai . This seems to be fixed in the latest version (`0.10.5`) - please report here if the problem persists. Thanks!
